core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreaker.java

```diff
@@ -73,4 +73,10 @@ static CircuitBreaker ofDefaultName() {
      * Decides whether a request should be sent or failed depending on the current circuit state.
      */
     boolean canRequest();
+
+    /**
+     * Transitions to the specified {@link CircuitState}.
```

Comment:
'Transition' is a noun. How about 'Enters the specifies ...'?

Reply:
Updated the documentation to match the method name 😅